docs/UserGuide.md

```diff
@@ -249,23 +249,48 @@ _Details coming soon ..._
 
 --------------------------------------------------------------------------------------------------------------------
 
-## FAQ
+## [FAQ](#faq)
+
+**Q**: What operating systems does HealthSync support?<br>
+**A**: HealthSync is a cross-platform application and supports Windows, macOS, and Linux. Simply follow the
+[installation guide](#quick-start) and download the jar file to use the application.
+
+**Q**: What are the advantages of using a CLI interface in HealthSync over traditional GUI apps?<br>
+**A**: HealthSync's CLI interface offers faster data entry and navigation for users who are comfortable with typing.
+It streamlines tasks and provides a more efficient way to manage patient details.
+
+**Q**: Can multiple users access Healthysync simultaneously, and how do I set up user accounts?<br>
+**A**: Healthysync is currently an application for a single user. Hence, it cannot be accessed simultaneously
```

Comment:
```suggestion
**A**: HealthSync is currently an application for a single user. Hence, it cannot be accessed simultaneously
```

Reply:
I agree with the change, committing the suggestion.